composer.json

```diff
@@ -27,10 +27,10 @@
         "behat/mink": "^1.7",
         "behat/mink-extension": "^2.1",
         "silverstripe/mink-facebook-web-driver": "^1",
-        "symfony/dom-crawler": "^3",
+        "symfony/dom-crawler": "^3|^4",
```

Comment:
Even though this works, standard syntax is "^3 || ^4"

https://getcomposer.org/doc/articles/versions.md#version-range

https://github.com/silverstripe/silverstripe-asset-admin/blob/1/composer.json#L9

Reply:
```suggestion
        "symfony/dom-crawler": "^3||^4",
```